### ISSUE TITLE
NPC Trader Error - Inventory randomly wipes

### ISSUE THREAD

**Issue by notcam45** [CitizensDev/Citizens#980]
The inventories of NPC's wipes every 10 or 15 minutes, I don't know why buts its a serious issue.

**Comment by Phealoon** [comment]
Yes we are having the SAME problem HUGE ISSUE for our RPG server.

**Comment by notcam45** [comment]
It's a glitch, if a player right click on the trader's inventory item and then closes their inventory right after right clicking it gives that player the item for free and removes it from the NPC's iinv.
